glue/cirq/stimcirq/_cirq_to_stim_test.py

```diff
@@ -463,16 +478,15 @@ def test_stimcirq_tags():
         ).with_tags("my_tag")
     )
 
-    stim_circuit = stimcirq.cirq_circuit_to_stim_circuit(c)
+    stim_circuit = stimcirq.cirq_circuit_to_stim_circuit(c, custom_op_conversion_func=_op_conversion)
     assert stim_circuit == stim.Circuit(
         """
-
- REPEAT 3 {
-        H 0
-        X 1
-        TICK
-        M 0 1
-        TICK
-    }
-"""
+        REPEAT 3 {
+                H 0
```

Comment:
Do a test where the stim circuit ends up containing a tag.

Reply:
@Strilanc this required some extra functionality (persisting cirq tags through to stim tags) beyond our use case here but I gave a stab at it. See my most recent commit for an example of what I'm thinking. I haven't run/written tests on it yet.